packages/@react-aria/table/src/useTableColumnResize.ts

```diff
@@ -76,45 +81,108 @@ export function useTableColumnResize<T>(props: AriaTableColumnResizeProps<T>, st
         }
       }
     },
-    onMoveEnd({pointerType}) {
-      if (pointerType !== 'keyboard') {
-        stateRef.current.onColumnResizeEnd(item);
-      }
+    onMoveEnd() {
       columnResizeWidthRef.current = 0;
       document.body.style.cursor = cursor.current;
     }
   });
+  let min = Math.floor(stateRef.current.getColumnMinWidth(item.key));
+  let max = Math.floor(stateRef.current.getColumnMaxWidth(item.key));
+  if (max === Infinity) {
+    max = Number.MAX_SAFE_INTEGER;
+  }
+  let value = Math.floor(stateRef.current.getColumnWidth(item.key));
 
   let ariaProps = {
-    role: 'separator',
     'aria-label': props.label,
-    'aria-orientation': 'vertical',
-    'aria-labelledby': item.key,
-    'aria-valuenow': stateRef.current.getColumnWidth(item.key),
-    'aria-valuemin': stateRef.current.getColumnMinWidth(item.key),
-    'aria-valuemax': stateRef.current.getColumnMaxWidth(item.key)
+    'aria-orientation': 'horizontal' as 'horizontal',
+    'aria-labelledby': `${id} ${getColumnHeaderId(state, item.key)}`,
+    'aria-valuetext': stringFormatter.format('columnSize', {value}),
+    min,
+    max,
+    value
+  };
+
+  const focusInput = useCallback(() => {
+    if (ref.current) {
+      focusWithoutScrolling(ref.current);
+    }
+  }, [ref]);
+
+  let onChange = (e: ChangeEvent<HTMLInputElement>) => {
+    let currentWidth = stateRef.current.getColumnWidth(item.key);
+    let nextValue = parseFloat(e.target.value);
+
+    if (nextValue > currentWidth) {
+      nextValue = currentWidth + 10;
+    } else {
+      nextValue = currentWidth - 10;
+    }
+    stateRef.current.onColumnResize(item, nextValue);
+  };
+
+  let onDown = () => {
+    focusInput();
```

Comment:
I see that it is like this in Slider and was there since the original implementation 2 years ago, just kinda feels weird to focus the visually hidden input when interacting via mouse/touch but maybe it is required for something

Reply:
yeah, this was required for some reason, I forget what now since it's been a few weeks. I'll take a look today and see if I can answer better

Reply:
Ok, it's used to make sure that we don't lose the blue focus color if the mouse leaves and is dragging not directly over it.